HTMLtask1/templ1.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Ellipsis</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" media="screen" href="templStyle1.css" />
+</head>
+<body>
+  <header class="main-page">
+      <div class="main">
+        <img src="pics/mainLabel.png" alt="Triangle label" id="triangleM">
+        <div class="main-head">Working<br> with design lab</div>
+        <div class="main-par">We’ve been helping startups launch and grow their businesses.<br>
+           We help you to fill the gap between development,<br>
+          design and product management.
+        </div>
+        <img src="pics/mainMouseLabel.png" alt="Scroller" id="scroller">
+      </div>
+  </header>
+  <section class="about">
+    <div class="about-head">About us</div>
+    <div class="about-par">When you work with us, you are part of a team of committed collaborators.<br> 
+      All aspects of our software development process are integrated, iterative and agile.<br><br>
+      Our approach unifies design, development and<br> 
+      product management to create exceptional products.<br> 
+      We start each project by understanding your business goals,<br> 
+      the user’s needs and the product requirements.<br> 
+      Then we help translate them into effective and elegant solutions.
+    </div>
+  </section>
+  <section class="services">
+    <div class="serv-head">Services</div>
+    <div class="serv-par">Before we sign a contract or write a line of code our team will spend the necessary<br> time needed to understand your divroduct vision.<br> 
+      The outcome of this scoping session is a proposal or high-level statement of work.<br> 
+      Based on your requirements, we can provide a complete offering or just the pieces<br> you need. </div>
+    </section>
+  <section class="what-de-do">
+    <div class="what-we-do-head">What we do</div>
+    <div class="text-block">
+      <div class="item">
+        <img src="pics/wwdDes.png" alt="Design" class="wwd-pic">
+        <div class="wwd-head">Design</div>
```

Comment:
don't be afraid to use long class names, eg. `what-we-do-head`. But in your case it's already used, so u can consider `what-we-do-item-head`. It's clean to read and understand